src/model/wallet.ts

```diff
@@ -330,7 +361,7 @@ export const signTransaction = async ({
   error?: any;
 }> => {
   try {
-    logger.sentry('about to sign transaction', transaction);
+    logger.info('wallet: signing transaction', { transaction });
```

Comment:
PII?

Reply:
this one u are right will strip out the payload